frontend/src/pages/Login/Login.jsx

```diff
@@ -2,47 +2,80 @@ import React from "react";
 import { Button, Pane, Heading, Strong, TextInputField } from "evergreen-ui";
 import "./style.css";
 
-function LoginPage(props) {
-    return (
-        <Pane
-            background="tint1"
-            height="100%"
-            display="flex"
-            alignItems="center"
-            justifyContent="center">
+
+class LoginPage extends React.Component {
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            email: "",
+            password: "",
+        };
+    }
+    handleClick = (val) => {
+        fetch("http://34.210.153.98:3000/user/login", {
```

Comment:
We shouldn't this url as a string; we may create a constants.js and get these url from there.